pkgs/games/gamemode/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, meson, ninja, fetchFromGitHub, systemd, pkgconfig, dbus }:
+
+stdenv.mkDerivation rec {
+  name = "gamemode-${version}";
+  version = "1.4";
+
+  buildInputs = [ meson ninja pkgconfig systemd dbus ];
```

Comment:
`meson`, `ninja`, and `pkgconfig` should likely be `nativeBuildInputs`. I'm not sure about the other 2

Reply:
systemd is needed at runtime, not sure about dbus, that might be able to go into native as well
```
$ ldd ./results/gamemode/bin/gamemoded
        linux-vdso.so.1 (0x00007fffd2ff1000)
        libsystemd.so.0 => /nix/store/qj5mxxzcwksccq69zlq9n49d8j4nvpj4-systemd-242-lib/lib/libsystemd.so.0 (0x00007ff578f9c000)
```

Reply:
The shared libraries in the package do link against dbus.